README.md

```diff
@@ -14,13 +14,30 @@ To run the demo, one should firstly [install `docker`](https://docs.docker.com/g
 docker-compose up -d
 ```
 
-Once the compose is up, the Point Network contracts deployment will start. Unless the contracts get deployed, the Point Network nodes will wait for the contract addresses to appear. As soon as the addresses obtained the nodes will get started. At this point you may deploy the demo websites via the following command:
+Once the compose is up, the Point Network contracts deployment will start. Unless the contracts get deployed, the Point Network nodes waits for the contract addresses to appear. As soon as the addresses obtained the nodes get started. At this point you may deploy the demo websites via the following command:
 
 ```bash
 ./scripts/deploy-sites-docker.sh
 ```
 
-Right after the sites are there, one may start the [Point Browser](https://github.com/pointnetwork/pointbrowser) and configure it to use one of the above listed `z-proxy` port. The sites will be available at their regular addresses.
+Right after the sites are uploaded, one may start the [Point Browser](https://github.com/pointnetwork/pointbrowser) and configure it to use one of the above listed `z-proxy` ports. The sites will be available at their regular addresses.
+
+### Develop using the docker compose
+
+If one needs to leverage the Point Network docker-compose and still be able to make changes in the code, they can do so by starting the compose the following way:
+
+```bash
+docker-compose -f docker-compose.yaml -f docker-compose.dev.yaml up -d
```

Comment:
hello @tankakatan i think there's a typo here ```docker-compose -f docker-compose.dev.yaml up -d```

Reply:
Thanks @Tunji17. As far as I get from the [docs](https://docs.docker.com/compose/extends/#understanding-multiple-compose-files), to extend one docker-compose file with another, one should specify both (or more) files in order from the lowest to the highest priority when running `docker-compose up`.

> To use multiple override files, or an override file with a different name, you can use the -f option to specify the list of files. Compose merges files in the order they’re specified on the command line. See the docker-compose command reference for more information about using -f.

Please let me know if I misunderstood your comment.